src/OneExplorer/ConfigObject.ts

```diff
@@ -214,37 +216,10 @@ export class ConfigObj {
   ): Artifact[] => {
     const dir = path.dirname(filePath);
 
-    let locatorRunner = new LocatorRunner();
-
-    locatorRunner.register({
-      artifactAttr: {
-        ext: ".tflite",
-        icon: new vscode.ThemeIcon("symbol-variable"),
-      },
-      locator: new Locator((value: string) =>
-        LocatorRunner.searchWithExt(".tflite", value)
-      ),
-    });
-
-    locatorRunner.register({
-      artifactAttr: {
-        ext: ".pb",
-        icon: new vscode.ThemeIcon("symbol-variable"),
-      },
-      locator: new Locator((value: string) =>
-        LocatorRunner.searchWithExt(".pb", value)
-      ),
-    });
-
-    locatorRunner.register({
-      artifactAttr: {
-        ext: ".onnx",
-        icon: new vscode.ThemeIcon("symbol-variable"),
-      },
-      locator: new Locator((value: string) =>
-        LocatorRunner.searchWithExt(".onnx", value)
-      ),
-    });
+    let locatorRunner = OneConfigSetting.getBaseModelsLocatorRunner();
+    if (path.extname(filePath) === ".edgetpucfg") {
```

Comment:
Where does it check if the edgetpu backend is registered or not?

Reply:
I temporarily check just filename. I will add the function that check if the edgetpu backend is registered or not soon.